copilot/fsd-frontend/manifest.yml

```diff
@@ -45,39 +45,58 @@ network:
 #
 # Pass environment variables as key value pairs.
 variables:
-  SENTRY_DSN: "https://3c6d2fd1e3824aecb3826a7d640b29a9@o1432034.ingest.sentry.io/4503897449103360"
-  FLASK_ENV: ${COPILOT_ENVIRONMENT_NAME}
-  COOKIE_DOMAIN: ".test.levellingup.gov.uk"
-  APPLICANT_FRONTEND_HOST: "https://frontend.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
-  AUTHENTICATOR_HOST: "https://fsd:fsd@authenticator.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk" # pragma: allowlist secret
   ACCOUNT_STORE_API_HOST: "http://fsd-account-store:8080"
+  APPLICANT_FRONTEND_HOST: "https://frontend.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
   APPLICATION_STORE_API_HOST: "http://fsd-application-store:8080"
-
-  FORMS_SERVICE_PUBLIC_HOST: "https://fsd:fsd@forms.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk" #pragma: allowlist secret
-  FORMS_SERVICE_PRIVATE_HOST: "https://fsd:fsd@forms.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk" #pragma: allowlist secret
-
+  AUTHENTICATOR_HOST: "https://authenticator.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk" # pragma: allowlist secret
```

Comment:
No need for `# pragma: allowlist secret` since we don't have a secret in there anymore.

Reply:
Fair point - removing